web/pdf_link_service.js

```diff
@@ -192,6 +192,13 @@ class PDFLinkService {
       destArray: explicitDest,
       ignoreDestinationZoom: this._ignoreDestinationZoom,
     });
+    const textLayerRenderedCallback = evt => {
+      if (evt.pageNumber === pageNumber) {
+        evt.source.textLayer.div.focus();
+        this.eventBus._off("textlayerrendered", textLayerRenderedCallback);
+      }
+    };
+    this.eventBus._on("textlayerrendered", textLayerRenderedCallback);
```

Comment:
Instead of the separate `textLayerRenderedCallback` variable we could also just inline it here since it's short enough.